src/ciphers/morse_code.rs

```diff
@@ -172,12 +172,7 @@ mod tests {
     #[test]
     fn decrypt_valid_character_set_invalid_morsecode() {
         let expected = format!(
-            "{}{}{}{} {}",
-            _UNKNOWN_MORSE_CHARACTER,
-            _UNKNOWN_MORSE_CHARACTER,
-            _UNKNOWN_MORSE_CHARACTER,
-            _UNKNOWN_MORSE_CHARACTER,
-            _UNKNOWN_MORSE_CHARACTER,
+            "{_UNKNOWN_MORSE_CHARACTER}{_UNKNOWN_MORSE_CHARACTER}{_UNKNOWN_MORSE_CHARACTER}{_UNKNOWN_MORSE_CHARACTER} {_UNKNOWN_MORSE_CHARACTER}",
```

Comment:
The original version seems more legible to me, but it's up to you

Reply:
Both are _ugly_. If the `_UNKNOWN_MORSE_CHARACTER` would be shorted, the 2nd version would look nicer. I do not see a better solution at the moment. I will merge it for not. Thanks for the review.